proto/shaman.proto

```diff
@@ -227,6 +227,7 @@ message EnhancementShaman {
 	message Rotation {
 		ShamanTotems totems = 1;
         bool lavaburst_weave = 2;
+		int32 weave_latency = 3;
```

Comment:
might need to adjust this, just merged in a pull request that changes some stuff in the proto file, not really anything too big though just double check it doesnt conflict

Reply:
No worries I don't mind conflicts :P